db/structure.sql

```diff
@@ -9,6 +9,13 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
+--
+-- Name: public; Type: SCHEMA; Schema: -; Owner: -
+--
+
+-- *not* creating schema, since initdb creates it
+
+
```

Comment:
There shouldn't be any changes to db/structure.sql in this PR

Reply:
> There shouldn't be any changes to db/structure.sql in this PR

This (changing db/structure.sql) started happening after #5157 and it occurs during running migration script (when configuring Docker). It's probably caused by different versions of DB in Ubuntu 22_04 and Debian 12.7 (different settings, initializations, ...).

Shall we silently commit this change in this PR, in other PR or should we deeper investigate it (or discard)?

Reply:
Removed changes to this file in the PR. Thank you